pkg/compiler/compilation_document.go

```diff
@@ -66,6 +90,25 @@ func (doc *CompilationDocument) OutputTo(dest string) error {
 	return nil
 }
 
+func postCompileHook(dir string, file core.File, hook string, log *zap.SugaredLogger) error {
+	hookSegments := strings.Split(hook, " ")
```

Comment:
Consider using [shlex](https://pkg.go.dev/github.com/google/shlex) to handle things like quotations.

Reply:
Ah cool, thanks for that library. That's a good call, because then it also lets people pass in a command of `bash -c 'whatever {} && whatever-else {}'` if they want